packages/kit/src/exports/index.js

```diff
@@ -84,7 +84,12 @@ const encoder = new TextEncoder();
 export function text(body, init) {
 	const headers = new Headers(init?.headers);
 	if (!headers.has('content-length')) {
-		headers.set('content-length', encoder.encode(body).byteLength.toString());
+		const encoded = encoder.encode(body);
+		headers.set('content-length', encoded.byteLength.toString());
+		return new Response(encoded, {
```

Comment:
is this a drive-by fix?

Reply:
Another perf thing to avoid duplicating the encoding work inside the Response constructor.